### PR TITLE
Fixes on license

### LINKED ISSUE #1: Failure to adhere to MIT License
Hey there 👋

This code was copied from [my repo](https://github.com/shirts/microsoft-teams-ruby), that you are a contributor of, without preserving the copyright.

https://github.com/shirts/microsoft-teams-ruby

Would appreciate a resolution on your end, new versions pushed to RubyGems etc. 

https://opensource.stackexchange.com/questions/8853/can-i-change-the-copyright-holder-name-and-year-of-an-mit-license
https://opensource.stackexchange.com/questions/11806/what-to-do-if-someone-forked-your-project-and-removed-its-license

Thanks.

### PR DESCRIPTION
**Pull request by pedrofurtado**
Closes #1 

@shirts 🤝 🍻

### CODE REVIEW

File: microsoft_teams_incoming_webhook_ruby.gemspec [hunk 1]
```diff
@@ -5,7 +5,7 @@ require_relative 'lib/microsoft_teams_incoming_webhook_ruby/version'
 Gem::Specification.new do |spec|
   spec.name                        = 'microsoft_teams_incoming_webhook_ruby'
   spec.version                     = MicrosoftTeamsIncomingWebhookRuby::VERSION
-  spec.authors                     = ['Pedro Furtado']
+  spec.authors                     = ['Kevin Chandler', 'Pedro Furtado']
```
Comment (shirts) [inline microsoft_teams_incoming_webhook_ruby.gemspec]: No need to add as author

**Review by shirts — DISMISSED** [review]
Thank you

**Comment by pedrofurtado** [comment]
@shirts Can you please review again, to ensure everything is ok? ✌️ 

I will release, after merge this PR, a new version of gem in rubygems containing this changes. 

Thank you so much for your hard work on original gem, because of it this project was possible to exists 🎉 

And sorry one more time for any inconvenience that I may be produced about licenses and copyrights, the origin was ignorance at my part in this subject 🤝

**Review by shirts — APPROVED** [review]

**Comment by shirts** [comment]
It's all good. Appreciate you

**Comment by pedrofurtado** [comment]
v1.0.2 released 🥇

**Pull request merged**